### PR TITLE
Dev v3 cc7

### PR DESCRIPTION
**Pull request by mexanick**
Working revision for V3 electronics with added code for xtalk measurements. Not compatible with V2 electronics. 
## Description

Also continues spurious commits due to `develop` branch of the central repo is **very** old
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context

Required to run the V3 electronics at the QC8 and integration stands
## How Has This Been Tested?

Compiled and ran under `gemuser` account at `gem904qc8daq` PC.

### CODE REVIEW

File: settings.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cd $GEM_DQM_ROOT_BASE
+source bin/thisroot.sh
+cd -
+export LD_LIBRARY_PATH=/usr/local/gcc/4.9.3/lib64:$LD_LIBRARY_PATH
```
Comment (jsturdy) [inline settings.sh]: is this still needed? i.e., is `4.8.x` too old?

File: settings.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+export PATH=$PATH:$BUILD_HOME/gem-light-dqm/dqm-root/bin/linux/x86_64_slc6:$BUILD_HOME/gem-light-dqm/gemtreewriter/bin/linux/x86_64_slc6
```
Comment (jsturdy) [inline settings.sh]: this has some `slc6` stuff leftover, is it because of a non-generic `Makefile`?

**Review by jsturdy — COMMENTED** [review]
Is this to remove support for V2, or be a placeholder for prototyping the V3/V2 compatible unpacker/plotter?
Also a couple of comments on some legacy stuff that seems to be sticking around.

File: settings.sh (hunk 2 above)
Comment (mexanick) [reply settings.sh]: Oups, actually this file can be safely removed at all. Not needed

**Comment by mexanick** [comment]
This is to have a placeholder for V3 development. The V2 legacy can sourced from the other branch and no new developments is foreseen for it (eventually the support will be dropped as we won't have V2 detectors...)

**Review by jsturdy — APPROVED** [review]
:+1:

**Pull request merged**